json/fr.json

```diff
@@ -106,7 +106,7 @@
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Veuillez confirmer l'accès à votre compte en entrant le code d'authentification fourni par votre application d'authentification.",
     "Please confirm your password before continuing.": "Veuillez confirmer votre mot de passe avant de continuer",
     "Please copy your new API token. For your security, it won't be shown again." : "Veuillez copier votre nouveau token API. Pour votre sécurité, il ne sera pas ré-affiché.",
-    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices." : "Veuillez entrer votre mot de passe pour confirmer que vous voulez déconnecter toutes les autres sessions navigateur sur l'ensemble de vos appareils.",
```

Comment:
Keys used in Laravel products cannot be deleted.

See: https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L112-L113

Reply:
You are right, but there are still some errors in the keys. Some are not present in fr.json 
Example with "logout" : 
 - https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L79-L80
 - https://github.com/Laravel-Lang/lang/blob/master/json/fr.json#L81

The "Log Out" variant used by Jetstream is not present in fr.json

Reply:
You did not understand. What I'm saying is that you have removed the option from "logout", although you need to leave both.